eurito_daps/packages/utils/openaire_utils.py

```diff
@@ -0,0 +1,165 @@
+from bs4 import BeautifulSoup
+import re
+from eurito_daps.production.orms import openaire_orm
+from eurito_daps.packages.utils import globals

```

Comment:
globals aren't recommended in python (also `globals` is a built-in method so expect some unexpected behaviour). I'd rewrite to avoid this!